core/frontend/src/tile/RealityModelTileTree.ts

```diff
@@ -821,9 +822,14 @@ export class RealityModelTileClient {
   public readonly rdsProps?: RDSClientProps; // For reality data stored on PW Context Share only. If undefined then Reality Data is not on Context Share.
   private _realityData?: RealityData;        // For reality data stored on PW Context Share only.
   private _baseUrl: string = "";             // For use by all Reality Data. For RD stored on PW Context Share, represents the portion from the root of the Azure Blob Container
-  private static _client = new RealityDataClient();  // WSG Client for accessing Reality Data on PW Context Share
   private _requestAuthorization?: string;      // Request authorization for non PW ContextShare requests.
 
+  private getInitializedRealityData(): RealityData {

```

Comment:
This while class concerns me a bit due to it assuming that the RealityDataClient is always there and the plan (based on the comment above to refactor). Not sure if it's in the scope of this PR but we should re-visit soon.

Reply:
We can clean this up in a subsequent PR.